testsuite/testsuite-microprofile-jwt/src/test/java/org/wildfly/swarm/microprofile/jwtauth/RolesAllowedNoPropsTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *   Copyright 2018 Red Hat, Inc, and individual contributors.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package org.wildfly.swarm.microprofile.jwtauth;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.security.KeyFactory;
+import java.security.PrivateKey;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Base64;
+
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.UrlAsset;
+import org.jboss.shrinkwrap.impl.base.io.IOUtil;
+import org.jose4j.jws.AlgorithmIdentifiers;
+import org.jose4j.jws.JsonWebSignature;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.Swarm;
+import org.wildfly.swarm.arquillian.CreateSwarm;
+import org.wildfly.swarm.jaxrs.JAXRSArchive;
+
+@RunWith(Arquillian.class)
+public class RolesAllowedNoPropsTest {
+
+    @Deployment
+    public static Archive<?> createDeployment() throws Exception {
+        JAXRSArchive deployment = ShrinkWrap.create(JAXRSArchive.class);
+        deployment.addResource(RolesEndpointClassLevel.class);
+        deployment.addResource(TestApplication.class);
+        deployment.addAllDependencies();
+        deployment.addAsResource("project-no-roles-props.yml");
+        URL url = RolesAllowedNoPropsTest.class.getResource("/publicKey.pem");
+        deployment.addAsManifestResource(new UrlAsset(url), "/MP-JWT-SIGNER");
+        return deployment;
+    }
+    @CreateSwarm
+    public static Swarm newContainer() throws Exception {
```

Comment:
Could remove the entire method if `project-no-roles-props.yml` was added to the archive under a name of `project-defaults.yml`?

Reply:
That would step over the default config which lists the empty properties ?

Reply:
I see, I did not understand it first.
However, I tried 'addAsResource("project-defaults.yml", "project-no-roles-props.yml")' and removed CreateSwarm and that failed the test, I guess this is because the 2nd test uses its own "project-defaults.yml"